src/sage/rings/multi_power_series_ring_element.py

```diff
@@ -1910,7 +1910,7 @@ def exp(self, prec=infinity):
         are not yet implemented and therefore such cases raise an error::
 
             sage: g = 2 + f
-            sage: exp(g)
+            sage: exp(g)                                                                # needs sage.symbolic
```

Comment:
You can avoid `SR` with `g.exp()` here and likewise in the other examples.

Reply:
Actually this error does not come from going through SR but illustrates that exp(2) does not exist in the base ring.

2 of the other `# needs sage.symbolic` turned out to be unnecessary due to advances in modularization. Removed in 7dde1297f33